app/Resources/translations/default/messages.xlf

```diff
@@ -2960,4 +2960,86 @@ Comment: $this->list_modules_categories['search_filter']['name'] = $this->l('Sea
       </trans-unit>
     </body>
   </file>
+  <file original="modules/gamification/controllers/admin/AdminGamificationController.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="3d4aafb2eedeba2fbf92e852f0af745a">
```

Comment:
Hey @eternoendless, there is no repertory for this module in translation files, it is part of the job we will have to do this summer with @colinegin to redefine modules' translation. :-)

@jolelievre, the following strings should not be localized in the _messages.xlf_ file, although they currently are. Would it be possible to create a dedicated folder, like _Modules.Merchantexpertise.Admin_? If it is not possible right now, we can do that later, no worries.

Reply:
Well it is possible, but it requires to update the module as well with the appropriate domain (same problem as other keys present in `messages` and `Admin.Actions`). I think this requires a large cleanup of the core code as well as native modules.